README.md

```diff
@@ -15,19 +15,19 @@ cross-platform way. Initially, it will provide a streamlined experience for deve
 ## Documentation
 
 * [Getting started with web-ext][web-ext-user-docs]
-* [Command reference](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/web-ext_command_reference)
+* [Command reference](https://extensionworkshop.com/documentation/develop/web-ext-command-reference)
 
 Here are the commands you can run. Click on each one for detailed documentation or use `--help` on the command line, such as `web-ext build --help`.
 
-* [`run`](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/web-ext_command_reference#web-ext_run)
+* [`run`](https://extensionworkshop.com/documentation/develop/web-ext-command-reference#web-ext-run)
   * Run the extension
-* [`lint`](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/web-ext_command_reference#web-ext_lint)
+* [`lint`](https://extensionworkshop.com/documentation/develop/web-ext-command-reference#web-ext-lint)
   * Validate the extension source
-* [`sign`](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/web-ext_command_reference#web-ext_sign)
+* [`sign`](https://extensionworkshop.com/documentation/develop/web-ext-command-reference#web-ext-sign)
   * Sign the extension so it can be installed in Firefox
-* [`build`](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/web-ext_command_reference#web-ext_build)
+* [`build`](https://extensionworkshop.com/documentation/develop/web-ext-command-reference#web-ext-build)
   * Create an extension package from source
-* [`docs`](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/web-ext_command_reference#web-ext_docs)
+* [`docs`](https://extensionworkshop.com/documentation/develop/web-ext-command-reference#web-ext-docs)
```

Comment:
This link is not working, because the `<section id...>` element is missing here: https://github.com/mozilla/extension-workshop/blame/29f868acaa3f7b804699fe95135f4559aebeb142/documentation/develop/web-ext-command-reference.md#L83

Do you want to submit a pull request in that repository to fix the documentation?

Reply:
Yes, I noticed it. But decided to leave the anchor in case the `<section id...>` element  will be added later.

Reply:
Related PR: https://github.com/mozilla/extension-workshop/pull/648